install/recorders.csv

```diff
@@ -1485,6 +1485,7 @@ Nanometrics,Titan,TitanSMA,002203,PNMS,21,283,,0,0,2024-02-29T02:18:00Z,9999-01-
 Nanometrics,Titan,TitanSMA,002204,FXBS,20,145,,0,0,2024-02-28T23:26:00Z,9999-01-01T00:00:00Z
 Nanometrics,Titan,TitanSMA,002205,WKZ,21,0,,0,0,2024-02-20T21:06:00Z,9999-01-01T00:00:00Z
 Nanometrics,Titan,TitanSMA,002208,WBHS,21,0,,0,0,2024-02-28T23:23:00Z,9999-01-01T00:00:00Z
+Nanometrics,Titan,TitanSMA,002209,TMHS,21,0,,0,0,2024-05-30T23:32:00Z,9999-01-01T00:00:00Z
```

Comment:
think its missing the new azimuth  of 250 here...

Reply:
Good catch, fixed